frontend/features/page/sections/FeatureListSection/FeatureListSectionBlock.tsx

```diff
@@ -40,8 +39,10 @@ export function FeatureListSectionBlock({
     };
   }, [preview]);
 
+  if (sectionTitle != null) sectionTitle = sectionTitle.toUpperCase();
```

Comment:
We try to always use curly brackets to avoid ambiguity

```suggestion
  if (sectionTitle != null) {
    sectionTitle = sectionTitle.toUpperCase();
  }
```